### PR TITLE
Feature#41/carousel products

### PR DESCRIPTION
**Pull request by germanruzca**
# Description
 - Create the partial of the card.
 - Add tiny styles in the card.
 - Create the corusel of the cards.
 - Add the buttons to show the cards when is clicked.
 
 - What?: I've added the corusel of the products.
 - Why?: To show the newest or older products to the user.
 See #41 for more context.
  
  ## Screenshots
carousel

### CODE REVIEW

**Review by juperameza — APPROVED** [review]
LGTM !

**Review by AlejandroCuriel — DISMISSED** [review]
LGTM :rocket: <3

**Review by nallely64 — DISMISSED** [review]
LGTM!

File: app/assets/stylesheets/components/_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  justify-content: space-between;
+}
+
+.carousel-control-prev, .carousel-control-next {
+  width: 30px;
+  margin: 0 10px;
+
+  i {
+    font-size: 40px;
+    color: #989696;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_carousel.scss:13]: change this value color for a variable

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/posts/_post_card.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <i class="fa-solid fa-star star"></i>
+    <i class="fa-solid fa-star star"></i>
+    <i class="fa-solid fa-star star"></i>
+    <i class="fa-solid fa-star-half star"></i>
+  </div>
+  <img src="" alt="">
+  <p class="card--info"><%= post.description %> </p>
+
+  <div class="card__more-info">
+    <%= link_to "Ver mas", "/desconocido"%>
```
Comment (YilKanda) [inline app/views/posts/_post_card.html.erb:24]: Is the path /desconocido functional?

File: app/assets/stylesheets/components/_carousel.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  }
+}
+
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_carousel.scss:16]: Remove the extra space.

File: app/assets/stylesheets/components/_carousel.scss (hunk 1 above)
Comment (germanruzca) [reply app/assets/stylesheets/components/_carousel.scss]: Done!, thanks

File: app/views/posts/_post_card.html.erb (hunk 2 above)
Comment (germanruzca) [reply app/views/posts/_post_card.html.erb]: No, I forgot to deleted it. New path added. Thanks

File: app/assets/stylesheets/components/_carousel.scss (hunk 3 above)
Comment (germanruzca) [reply app/assets/stylesheets/components/_carousel.scss:16]: Done!

**Review by juperameza — APPROVED** [review]
LGTM!

**Review by Javi1499 — APPROVED** [review]
LGTM

**Comment by JuanequeX** [comment]
Solve the conflict of your branch

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**